src/fibonacciBases.js

```diff
@@ -0,0 +1,48 @@
+function convertToFib() {
+  let number = document.getElementById('number').value
+  let result = []
+  let fibSeq = []
```

Comment:
Consider avoiding the use of abbreviations - complete words are more intention revealing, and more likely to be understood by developers reviewing or maintaining this code.